### DISCUSSION TITLE
Costs calculation running AzuraCast for +2000 listeners

### DISCUSSION THREAD

**Discussion by esturniolo** [AzuraCast/AzuraCast#6367] [Q&A]
Hi guys. I understand that this question is not 100% related with AzuraCast, but I don't realize how to calculate the following.

I want to run AzuraCast for a station with the following requirements:
- 2 live shows a day (3 hours each) with an average of 2000 listeners
- Another 500 (avg) listener that hear the same shows later using the Podcast section. Each show has around 180MB

I don't know how calculate the costs for these kind of usage. So without knowing this I can't try to find the best VPS provider for this use case.
Someone of you has similar needs? If so, can you share please your costs for this?  

Thanks in advance!!!

**Comment by robz0** [comment]
@esturniolo you could try something like this:
https://www.radioforge.com/internet-radio-bandwidth-calculator/
or this..
https://castr.com/bandwidth-calculator/

Both got from searches on 'radio streaming bandwidth calculator' or similar search term. 
Don't really know if they correlate to each other or to reality but it might give you an idea to start with.

The other you could do is try something - suck it and see - as they say. Unless you are already at 2000 listeners in which case it might be too much to test.
**However** Azuracast can easily be migrated to another server through it's backup system. So if you had to move server to save costs, it would be totally possible. Most you might lose is your current ip address.

Some VPS providers offer Unlimited Bandwidth, which might be worth considering as I guess that listener growth is the unpredictable variable cost.

Worth getting most affordable ram/cpu also as I guess 2000 listeners would make an impact on that also. 

Hope helps

**Comment by esturniolo** [comment]
Hi @robz0 thanks!!!!!
This is the kind of calculator that I want to find. I don't know why I didn't find them when I prompted my search.

Regardigns the VPS, yes, we should consider some kind of unlimited bandwith for this.

Finally, for this kind of usage, is enough to have only the bandwidth size in consideration (1Gbps at least) or I should need to keep in mind another hardware aspect in the VPS like RAM or CPU?

Thanks again!
